docs/formats/sly_pointcloud_user_manual.md

```diff
@@ -0,0 +1,183 @@
+# Supervisely Point Cloud user manual
+
+## Contents
+
+- [Format specification](#format-specification)
+- [Import Supervisely Point Cloud](#import-supervisely-point-cloud-dataset)
+- [Export to other formats](#export-to-other-formats)
+- [Export to Supervisely Point Cloud](#export-to-supervisely-point-cloud)
+- [Examples](#examples)
+
+## Format specification
+
+Point Cloud data format:
+- [specification](https://docs.supervise.ly/data-organization/00_ann_format_navi).
+- [example](https://drive.google.com/file/d/1BtZyffWtWNR-mk_PHNPMnGgSlAkkQpBl/view).
+
+This dataset format supports the following types of annotations:
+- `cuboid_3d`
+
+Supported annotation attributes:
+- `object` (read/write, integer),
+- `createdAt` (write, string),
+- `updatedAt` (write, string),
+- `labelerLogin` (write, string), responsible for the corresponding fields
+  in the annotation file.
+- arbitrary attributes
+
+Supported image attributes:
+- `description` (read/write, string),
+- `createdAt` (write, string),
+- `updatedAt` (write, string),
+- `labelerLogin` (write, string), responsible for the corresponding fields
+  in the annotation file.
+- `frame` (read/write, integer). Indicates frame number of the image.
+- arbitrary attributes
+
+## Import Supervisely Point Cloud dataset
+
+An example datast in Supervisely Point Cloud format is available for download:
+
+<https://drive.google.com/u/0/uc?id=1BtZyffWtWNR-mk_PHNPMnGgSlAkkQpBl&export=download>
```

Comment:
Where does this example come from?

Reply:
Actually, I don't know. @nmanovic, how did it get in SOW?